### ISSUE TITLE
Sound Quality - HD Audio

### ISSUE THREAD

**Issue by mfisher224** [BimmerGestalt/AAIdrive#452]
I just upgrade to the Pixel 6 Pro and when I connected bluetooth, the phone showed that the car is capable of HD Audio via AAC codec. Does AAIdrive support this? I now feel like when I switch from listening via straight bluetooth to listening via AAIdrive, the quality through AAIdrive isnt as good.

**Comment by hufman** [comment]
As far as I understand it, the exact same A2DP connection is reused for both straight Bluetooth and App Music connections. Your new phone's AAC support should just work, my Galaxy S9 also seems to run in AAC mode. I suspect the car may have different Volume and Tone settings for each input type (USB files, Radio, Bluetooth, App), and so you might check the Tone settings to see if they are different between the different modes.

**Comment by hufman** [comment]
Did you find out if there was a difference in the Tone settings between these two sources? The audio signal from the phone to the car is identical, the app isn't changing the Bluetooth connection settings when switching sources and Android is just blindly playing audio out through Bluetooth even if the app is controlling it, so it must be something different about the car's eq settings.

**Comment by mfisher224** [comment]
I don't think I can adjust tone by input, but it may be that I am imagining
a difference and its actually identical.

**Comment by hufman** [comment]
You are right, I misremembered. I thought the car had different volumes per source, and assumed the tone would be different per source, but I just went out to confirm and the volume and tone is the same for my radio and Bluetooth inputs. I apologize for the confusion!
Can you try to record a video showing the different in audio quality?
